ff2mpv.js

```diff
@@ -1,12 +1,17 @@
-function ff2mpv(url) {
-    browser.runtime.sendNativeMessage("ff2mpv", { url: url });
+function ff2mpv(url, opt) {
```

Comment:
Instead of a string, let's consider making `opt` a dictionary that we put a `video: bool` key into. In other words, the message should be:

```
{
    url: "...",
    opt: {
        video: true,
    }
}
```

That makes us more flexible, for future options.